src/ol_openedx_canvas_integration/settings/common.py

```diff
@@ -0,0 +1,8 @@
+"""Common settings unique to the canvas integration plugin."""
+
+
+def plugin_settings(settings):
+    """Settings for the canvas integration plugin."""
+    settings.CANVAS_ACCESS_TOKEN = None
+    settings.CANVAS_BASE_URL = None
+    settings.FEATURES['ENABLE_CANVAS_INTEGRATION'] = False
```

Comment:
By default this feature should be enabled.

Reply:
I agree with the default, but do we even need a feature flag? If the plugin is installed, shouldn't that be a clear signal that it should be turned on?

Reply:
We've been running the canvas behind a feature flag from when we first implemented that & I'm not sure what was the reason to keep it behind a flag but we can remove it and assume if the plugin is installed, the canvas is enabled.
The flag is used only once in the [edx counterpart](https://github.com/mitodl/edx-platform/pull/274/files#diff-0d1ee66fb84bee6c3517980a7cefd74a050ccb60da9c0c3b662ab0a2d80d17c0R151) and we can make that check `settings.INSTALLED_APPS` instead of a flag. I'll remove the feature flag now unless you say otherwise.

Reply:
I think it would be better to test on INSTALLED_APPS, but if you want to leave that for another issue & PR, I would be OK with that.

Reply:
Updated.